setup.py

```diff
@@ -20,4 +29,5 @@ def read(fname):
       long_description=read('README.rst'),
       long_description_content_type='text/x-rst',
       include_package_data=True,
-      license="BSD-3")
+      license="BSD-3",
+      classifiers=CLASSIFIERS)
```

Comment:
Would it make sense to add `python_requires=">=3.6"` [like we did in civis-python](https://github.com/civisanalytics/civis-python/blob/a14ecd545a2e01521606e570f3415b32e8908e7e/setup.py#L69)?

Reply:
ah, i forgot about that. thanks